scripts/pc2validate_interactive.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Usage is: copy_xml_result.sh {:resfile} {:feedbackdir} {:testcase}
```

Comment:
This comment doesn't make sense -- it describes a usage for a script named `copy_xml_result.sh`, which is not this script.

Reply:
Will be fixed in next commit.